modules/server_new/src/main/scala/observe/server/ObserveEngine.scala

```diff
@@ -402,45 +402,42 @@ object ObserveEngine {
     // Produce a Handle that will send a SequenceStart notification to the ODB, and produces the (sequenceId, stepId)
     // if there is a valid sequence with a valid current step.
     private def sequenceStart(
-      id: Observation.Id
+      obsId: Observation.Id
     ): HandlerType[F, Option[(Observation.Id, Step.Id)]] =
```

Comment:
I think we can simplify the signature here, since it's not triggering any further events anymore.